app/Http/Controllers/DashboardController.php

```diff
@@ -0,0 +1,122 @@
+
+        $countData = [
+            'user' => User::count(),
+            'enabledWebhook' => Webhook::enable()->count(),
+            'disabledWebhook' => Webhook::disable()->count(),
+            'bot' => Bot::count(),
+        ];
+
+        $period = $this->getDateFromRange($statisticParams['fromDate'], $statisticParams['toDate']);
+        $payloadFailded = PayloadHistory::dataChart($statisticParams, PayloadHistoryStatus::FAILED);
+        $payloadSuccess = PayloadHistory::dataChart($statisticParams, PayloadHistoryStatus::SUCCESS);
+        $messageFailed = MessageHistory::dataChart($statisticParams, MessageHistoryStatus::FAILED);
+        $messageSuccess = MessageHistory::dataChart($statisticParams, MessageHistoryStatus::SUCCESS);
+
+        $payloadFaildedQuantity = $this->getDataChart($payloadFailded, $period);
+        $payloadSuccessQuantity = $this->getDataChart($payloadSuccess, $period);
+        $messageFaildedQuantity = $this->getDataChart($messageFailed, $period);
+        $messageSuccessQuantity = $this->getDataChart($messageSuccess, $period);
+
+        $payloadFailedChart =  $this->convertArrayChart($payloadFaildedQuantity, $period);
+        $payloadSuccessChart = $this->convertArrayChart($payloadSuccessQuantity, $period);
+        $messageFailedChart = $this->convertArrayChart($messageFaildedQuantity, $period);
+        $messageSuccessChart = $this->convertArrayChart($messageSuccessQuantity, $period);
+        $dateChart = $this->convertArrayChart($period);
+
+        $payloadChartData = [
+            'failedCases' => array_sum($payloadFaildedQuantity),
+            'successCases' => array_sum($payloadSuccessQuantity),
+            'dateChart' => $dateChart,
+            'payloadFailedChart' => $payloadFailedChart,
+            'payloadSuccessChart' => $payloadSuccessChart,
+        ];
+        $messageChartData = [
+            'failedCases' => array_sum($messageFaildedQuantity),
+            'successCases' => array_sum($messageSuccessQuantity),
+            'dateChart' => $dateChart,
+            'messageFailedChart' => $messageFailedChart,
+            'messageSuccessChart' => $messageSuccessChart,
+        ];
+        return view(
+            'dashboard',
+            [
+                'countData' => $countData,
+                'payloadHistory' => $payloadChartData,
+                'messageHistory' => $messageChartData,
+            ]
+        );
+    }
+
+    private function handleStatisticParam($statisticParams) {
+        if ($statisticParams == null) {
+            $statisticParams = ['fromDate' => date('01-m-Y'), 'toDate' => date('d-m-Y')];
+        }
+
+        if ($statisticParams['fromDate'] == null) {
+            $statisticParams['fromDate'] = date('01-m-Y');
+        }
+
+        if ($statisticParams['toDate'] == null) {
+            $statisticParams['toDate'] = date('d-m-Y');
+        }
+        return $statisticParams;
+    }
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Http/Controllers/DashboardController.php</br>**Line**: 88</br>Opening brace should be on a new line